scripts/network.ergocub-2foc-special.xml

```diff
@@ -0,0 +1,259 @@
+<?xml version="1.0"?>
```

Comment:
Do you think is it possible to move this file in robots-configuration/ergocub001 and ergocub002, with a name like "network.$YARP_ROBOT_NAME.2focspecial.xml"? if someone tries to run this script first of all he/she gets the warning that the script works only on ergcub and then faces the request to continue.